frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -474,8 +474,25 @@ export const eventUsageLogic = kea<
                 properties.path_type = filters.path_type
                 properties.has_start_point = !!filters.start_point
                 properties.has_end_point = !!filters.end_point
-                properties.has_funnel_filter = !!filters.funnel_filter
+                properties.has_funnel_filter = Object.keys(filters.funnel_filter || {}).length > 0
                 properties.funnel_paths = filters.funnel_paths
+                properties.has_min_edge_weight = !!filters.min_edge_weight
+                properties.has_max_edge_weight = !!filters.max_edge_weight
+                properties.has_edge_limit = !!filters.edge_limit
+                properties.has_local_cleaning_filters = (filters.local_path_cleaning_filters || []).length > 0
+                properties.has_path_replacements = !!filters.path_replacements
+                properties.has_wildcards = (filters.path_groupings || []).length > 0
+                properties.using_advanced_features =
+                    properties.has_min_edge_weight ||
+                    properties.has_max_edge_weight ||
+                    properties.has_edge_limit ||
+                    properties.has_local_cleaning_filters ||
+                    properties.has_path_replacements
+                properties.using_basic_features =
```

Comment:
Nit: `using_basic_features` seems a confusing name because I would expect it to be the complementary of `using_advanced_features` (as in MECE). It looks like we would use this to define Paths that at least are using some filtering features but no advanced features right?

Reply:
Yes precisely, since they're not actually MECE. Specifically, they're not mutually exclusive: You can have both at the same time 🤷

Reply:
Haha exactly, I meant that the name makes me think it's mutually exclusive (as in you either use basic or advanced features), but the code is very clear. Seems like a good use case for taxonomy just to avoid the confusion.